dnf/yum/rpmsack.py

```diff
@@ -15,6 +15,7 @@
 from __future__ import absolute_import
 from __future__ import unicode_literals
 from . import misc
+from sys import version_info
```

Comment:
You can save two lines if you import it from pycomp (`from .pycomp import PY3`).